js/okx.js

```diff
@@ -2252,7 +2299,12 @@ module.exports = class okx extends Exchange {
         if (limit !== undefined) {
             request['limit'] = limit; // default 100, max 100
         }
-        const response = await this.privateGetTradeOrdersPending (this.extend (request, params));
+        let method = 'privateGetTradeOrdersPending';
+        const ordType = this.safeString (params, 'ordType');
```

Comment:
this should be `const stop = this.safeValue (params, 'stop');`

and then you should check `if (stop) { ...`

Reply:
I'd say, we should do both checks, `stop` (unified, boolean) and `ordType` (exchange-specific, string).